shared-libs/transitions/test/unit/transitions.js

```diff
@@ -333,5 +336,186 @@ describe('transitions', () => {
     assert.isDefined(deprecatedTransitions);
     assert.equal(deprecatedTransitions.length, 2);
   });
+
+  describe('applyTransition',  () => {
+    it('should apply transition with change', async () => {
+      const transition = {
+        key: 'mytransition',
+        filter: sinon.stub().returns(true),
+        onMatch: sinon.stub().resolves(true),
```

Comment:
Serious question (not trying to be "smart" here, but actually wanting to understand the criteria): is this a case where we should follow the [method chaining formatting guidelines](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/#method-chaining) and put these calls on separate lines?

Reply:
Method chaining on multiple lines is strictly for readability and to be consistent: 
So it's: 
- super short functions that fit in one line (eg `sinon,stub().withArgs(3).returns(6);`) and refer to one thing (semantically)
- two functions that fit well in one line (eg `array.map(value => value * 2).filter(value => value)`)
- every call on a separate line

Idea is that there is no inconsistency, with some functions inlined and some not:
```
thing.call()
  .call()
  .call()
```